src/libstd/panicking.rs

```diff
@@ -167,9 +167,9 @@ fn default_hook(info: &PanicInfo<'_>) {
         let panics = update_panic_count(0);
 
         if panics >= 2 {
-            Some(backtrace::PrintFormat::Full)
+            Some(backtrace::PrintFmt::Full)
```

Comment:
```suggestion
            Some(backtrace_mod::PrintFmt::Full)
```
? or is `backtrace` the crates.io crate and `backtrace_mod` the 'rustc' crate?

Reply:
This is coming from the `backtrace` crates.io crate. `std::backtrace` was imported here as `backtrace_mod` to distinguish between the two.